fe/fe-core/src/main/java/org/apache/doris/statistics/StatisticsRepository.java

```diff
@@ -100,6 +101,11 @@ public class StatisticsRepository {
                     + " ORDER BY update_time "
                     + "LIMIT ${limit} OFFSET ${offset}";
 
+    private static final String FETCH_STATS_PART_ID = "SELECT DISTINCT col_id, part_id FROM "
```

Comment:
I think `distinct` here is useless, since every part is unique

Reply:
thx, you're right